doc/changelog.md

```diff
@@ -0,0 +1,923 @@
+# Changelog
+
+Listed here are the changes between each release of SmartSim,
+SmartRedis and SmartDashboard.
+
+Jump to {ref}`SmartRedis changelog<SmartRedis changelog>` or {ref}`SmartDashboard changelog<SmartDashboard changelog>`
+
```

Comment:
I'm not 100% sure what these lines are supposed to be doing but they don't link or jump anywhere, just fyi! Let me know if I'm missing something though.

Reply:
here Im following MyST syntax to reference a section label using a role - so

```
(this-is-a-role)=
## Section title

I want the role to be included {ref}`here<this-is-a-role>`
```
so the word here will be linked to the Section title

Reply:
its similar to how you would do it in an rst file -> I want the role to be included :ref:`here<this-is-a-role>`

.. _this-is-a-role:
## Section title

Reply:
Okay that makes a lot of sense! Thank you for explaining it!